core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -30,6 +30,7 @@ import scala.util.Properties
 import org.apache.commons.lang3.StringUtils
 import org.apache.hadoop.fs.Path
 import org.apache.hadoop.security.UserGroupInformation
+import org.apache.hadoop.yarn.conf.YarnConfiguration
```

Comment:
Better to explicitly declare a dependency rather than rely on transitivity, right?